cmip6_downscaling/tests/methods/test_regions.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import xarray as xr
+
+from cmip6_downscaling.methods.regions import combine_outputs, generate_subdomains
+
+
+def example_3d_dataarray_us_domain(val=0):
```

Comment:
oooh i love this function- will def use it!! - can we move this to somewhere else for more general use?

Reply:
i put it in tests/fixtures, does that work?

Reply:
lovely ✨